README.md

```diff
@@ -163,11 +189,13 @@ const targetOptions = {
     html5AsBinary: true         // upload HTML5 templates as binary
 }
 
-gulp.task('deploy', function() {
-	return gulp.src('**/*', {cwd: 'build'})
+function deployNewer () {
+	return gulp.src('**/*', {cwd: '.'})
```

Comment:
Is this correct? No more building with ant

Reply:
Not necessarily ;)
What is ant capable of that gulp cannot do?
I do deploy to existdb via gulp-exist since years. Or is it the name of the function or the working directory?

Reply:
nvm `ant` it just strikes me as different behavior. I make `.xars` for deployment and deploy of there is a newer one. This now looks like its more like `gulp.watch` no?

Reply:
It is not a watcher but could be called by one. `watch('**/*', series(deployNewer))`